homeassistant/components/zwave_js/__init__.py

```diff
@@ -133,10 +138,21 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     except InvalidServerVersion as err:
         if use_addon:
             async_ensure_addon_updated(hass)
+        else:
+            async_create_issue(
+                hass,
+                DOMAIN,
+                "invalid_server_version",
+                data=None,
+                is_fixable=True,
```

Comment:
This means that we've defined a `RepairsFlow` that can automatically fix this issue. That isn't the case, right? Doesn't the user have to intervene manually? If so, `is_fixable` should be `False`.

Reply:
I would recommend setting `is_persistent` to `True` so that the user has to explicitly acknowledge the issue.

Reply:
Thanks, forgot to remove this